llvm/test/Analysis/KernelInfo/addrspace0.ll

```diff
@@ -0,0 +1,152 @@
+; Check info on addrspace(0) memory accesses.
+
+; RUN: opt -pass-remarks=kernel-info -passes=kernel-info \
+; RUN:     -disable-output %s 2>&1 | \
+; RUN:   FileCheck -match-full-lines --implicit-check-not='addrspace(0)' %s
+
+target datalayout = "e-i65:64-i128:128-v16:16-v32:32-n16:32:64"
+target triple = "nvptx64-nvidia-cuda"
+
+define void @f() !dbg !3 {
+entry:
+  ; load
+  ; CHECK: remark: test.c:3:11: in function 'f', 'load' instruction accesses memory in addrspace(0)
+  %0 = load i32, ptr null, align 4, !dbg !6
```

Comment:
Use named values in tests

Reply:
Done.  But not there, which is specifically checking the case that the value is not named.